launch.go

```diff
@@ -147,7 +147,7 @@ func quickLauncherHandler(w http.ResponseWriter, r *http.Request) {
 
 	urlValues.Add("ru_ref", authentication.GetDefaultValues()["ru_ref"])
 	urlValues.Add("collection_exercise_sid", uuid.NewV4().String())
-	urlValues.Add("case_id", uuid.NewV4().String())
```

Comment:
From @MebinAbraham :

Do we want to keep this anyway since it is optional now in runner.

Reply:
I think the answer is probably yes, since there is still runner code that uses case_id, and I don't think it fits as questionnaire metadata. May be useful to test with runner.

Reply:
I've added this back in for now. There should be unit tests within runner that cover submitter behaviour. I'm guessing once case_id is no longer part of the payload it can be removed entirely.